docs/getting-started/integration.md

```diff
@@ -84,3 +84,18 @@ require(['moment'], function() {
     });
 });
 ```
+
+## Content-Security-Policy
```

Comment:
I would probably leave out the hyphens in this text since it's uncommon to include them. The header name is `Content-Security-Policy` with hyphens, but when talking about the functionality and not the header itself hypens are not used